rustls/tests/common/mod.rs

```diff
@@ -218,6 +221,19 @@ impl KeyType {
             .collect()
     }
 
+    pub fn get_client_crl(&self) -> UnparsedCertRevocationList {
+        UnparsedCertRevocationList(
+            rustls_pemfile::crls(&mut io::BufReader::new(
+                self.bytes_for("client.revoked.crl.pem"),
+            ))
+            .unwrap()
+            .iter()
```

Comment:
If we `into_iter()` here instead, can we avoid the `clone()` at the end?

Reply:
Good call, fixed.